MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -58,22 +57,15 @@ class MultiPartyViewController: UIViewController {
     override func viewWillLayoutSubviews() {
         super.viewWillLayoutSubviews()
 
-        var layoutFrame = self.view.bounds
+        var layoutFrame = self.containerView.bounds
```

Comment:
Can the container view occupy some or all of the safe area reserved for buttons? This looks pretty odd in landscape.

![Simulator Screen Shot - iPhone 6 - 2019-04-18 at 12 35 01](https://user-images.githubusercontent.com/1302577/56386433-9445aa80-61d6-11e9-88bf-926bdfd2b27f.png)

Reply:
Hiding the nav bar helps, leaving the rest for later.